core/new-gui/src/app/dashboard/component/dashboard.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="dashboard-grid-container grid-container">
+	<texera-top-bar class="dashboard-top-bar grid-container"></texera-top-bar>
+	<texera-feature-bar class="dashboard-feature-bar grid-container"></texera-feature-bar>
+	<div class="dashboard-feature-box grid-container">
+			<h1>here is the feature container</h1>
+	</div>
```

Comment:
The feature container should be in its own component and be referenced in `dashboard.component.html`.

Reply:
Currently, in the second PR, I don't have feature container as a component. I'll have the component in next PR, or I can add an empty feature container component into this PR